packages/webapp/pages/notifications.tsx

```diff
@@ -34,6 +43,40 @@ const hasUnread = (data: InfiniteData<NotificationsData>) =>
     page.notifications.edges.some(({ node }) => !node.readAt),
   );
 
+const contextId = 'notifications-context-menu';
+
+interface ActionCopy {
+  mute: string;
+  unmute: string;
+}
+
+const actionCopy: Partial<Record<NotificationType, ActionCopy>> = {
+  [NotificationType.ArticleNewComment]: {
+    mute: 'Mute new comments',
+    unmute: 'Unmute new comments',
+  },
+  [NotificationType.SquadNewComment]: {
+    mute: 'Mute new comments',
+    unmute: 'Unmute new comments',
+  },
+  [NotificationType.CommentReply]: {
+    mute: 'Mute this thread',
+    unmute: 'Unmute this thread',
+  },
+  [NotificationType.SquadReply]: {
+    mute: 'Mute this thread',
+    unmute: 'Unmute this thread',
+  },
+  [NotificationType.SquadPostAdded]: {
+    mute: 'Mute new posts',
+    unmute: 'Unmute new posts',
+  },
+  [NotificationType.SquadMemberJoined]: {
+    mute: 'Mute new members joining',
+    unmute: 'Unmute new members joining',
+  },
+};
```

Comment:
Maybe extract this to common, just think it might be reused in other components/pages down the line.